careers.rst

```diff
@@ -206,6 +206,25 @@ They usually have strong skill overlaps with Network Engineers (with so much
 modern storage being network-attached in some manner) and usually have strong
 skills in capacity planning and performance management of infrastructure.
 
+High Performance Computing Engineer
+===================================
+
+High Performance Computing (HPC) involves large scale infrastructure,
+which is at the cutting edge of presently available technology.
+Typically, HPC platforms are used for Scientific Computing, Big Data, Complex Models
+and may have applications in fields as diverse as physics, finance, medicine etc.
+for instance: Meteorology and Climate prediction are just a tiny slice of the bigger picture.
+
+HPC engineers are expected to master an array of technologies in fields such as
+networking (InfiniBand, multi-Gigabit Ethernet), computing (several computer architectures),
+parallel storage/filesystems/filers (Lustre, GPFS, Isilon, NetApp, PanaSAS), as well as
+be able to give advice on a number of software components (gnu/intel/pgi compilers,
+debuggers, mpi stacks, linear algebra and other optimized math libraries etc etc).
+
+Most importantly, HPC engineers should be able to interface with other operations engineers,
+each specialist in her field, in order to let all systems run at top performance.
```

Comment:
"each specialist in her field" doesn't really make much sense here, and I think it'll be better with "their" rather than "her". How about "...operations engineers, each a specialist in their own field, in order to..."?